src/application/database/mongodb/users-mongo-repository.js

```diff
@@ -0,0 +1,22 @@
+import { MongoHelper } from '../../helpers/mongoHelper';
+
+export class UsersMongoRespository {
+  async create(userData) {
+    const usersCollection = await MongoHelper.getCollection('users');
+    const result = await usersCollection.insertOne(userData);
```

Comment:
Isso aqui é pro cadastro? Se for, ele verifica se os dados já existem?

Reply:
Não, isso é só um repositorio que permite abstrair as queries do banco. Com ele consigo alterar o banco de SQL pra NoSQL, ou mudar uma ORM, ODM sem grandes problemas